core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties

```diff
@@ -22,30 +22,31 @@
 
 blurb.1=\
   When there are changes in how data is stored on disk, Jenkins uses the following strategy: \
-  data is migrated to the new structure when it is loaded, but the file is not resaved in the \
-  new format.  This allows for downgrading Jenkins if needed.  However, it can also leave data \
-  on disk in the old format indefinitely.  The table below lists files containing such data, \
-  and the Jenkins version(s) where the data structure was changed.
+  data is migrated to the new structure when it is loaded, but the project is not resaved in \
```

Comment:
Not necessarily correct. _Item_ might take care of, well, items (including other job types, folders, etc.), but this often applies to build records.

Reply:
I agree with @daniel-beck. `project` is too close to "job" imo.